pokerole_irl/worlds_app/views.py

```diff
@@ -47,6 +46,22 @@ def form_valid(self, form):
         membership.save()
         return response
 
+class WorldUpdateView(LoginRequiredMixin, UpdateView):
+    template_name = "world_update.html"
+    model = World
+    fields = ("__all__")
+    context_object_name = "world"
+
+    def dispatch(self, request, *args, **kwargs):
+        self.world = World.objects.get(pk=self.kwargs.get("pk"))
+        if request.user.is_superuser or self.world.user == request.user:
+            return super().dispatch(request, *args, **kwargs)
+        else:
+            raise PermissionDenied("You are not the owner of this world.")
+    
+    def get_success_url(self) -> str:
+        return reverse_lazy('world', kwargs={"pk":self.object.pk})
```

Comment:
Consider putting this on the World-model as a `get_absolute_url`-method instead.

Reply:
done